rollup/rollup-cdn.config.js

```diff
@@ -45,8 +44,6 @@ export default {
   plugins: [
     // output directory cleanup
     clearPlugin({ outputDir }),
-    // transform .json files into ES6 modules. (Used by statuses library). TODO: we should get rid of this library
-    json(),
```

Comment:
Maybe I missed something here. I thought we needed that because we use the `statuses` dependency. We do not use it anymore?

Reply:
Oh, you mean this error? :rofl: 

It's not needed in one of our two builds but it's the other one. It should be in the CDN build but not in the npm build.

![Capture d’écran du 2022-08-26 10-32-39](https://user-images.githubusercontent.com/236342/186859571-c489dab5-1890-4e58-92b6-9efe991b485e.png)

Reply:
It's actually what the commit says :-( Thanks again for catching this. I fixed id.